EIPS/eip-7968.md

```diff
@@ -0,0 +1,370 @@
+---
+eip: 7968
```

Comment:
```suggestion
eip: 7557
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename & Eth Magicians post